cylc/flow/task_state.py

```diff
@@ -407,10 +407,6 @@ def reset(self, status=None, is_held=None):
         status, except for custom outputs on reset to succeeded or later -
         these can be completed if need be using "cylc reset --output".
```

Comment:
With the removal of cylc reset, can this  `- these can be completed if need be using "cylc reset --output".`  now go too?

Reply:
Yes.